src/_data/sidenav.yml

```
@@ -1,38 +1,65 @@
-- title: Get started
+- header: Get started
+- title: Installation
   children:
     - title: Install Flutter
       permalink: /get-started/install
     - title: Test drive
       permalink: /get-started/test-drive
+
+
+- title: Learn Flutter
+  children:
+    - title: Introduction
+      permalink: /get-started/learn-flutter
+    - title: Dart language overview
+      permalink: https://dart.dev/overview
     - title: Write your first app
       permalink: /get-started/codelab
-    - title: Learn more
-      permalink: /get-started/learn-more
-    - divider
-    - title: Where to get Flutter support
-      permalink: /resources/support
-    - title: From another platform?
-      permalink: /get-started/flutter-for
+    - title: Fundamentals course
```

As a note, if you want this content to be the most important and easiest to discover in this category, you can mark this section as `expanded: true` to start as open.

```suggestion
    - title: Fundamentals course
      expanded: true
```